### PR TITLE
Change Zap automation UI issues

### LINKED ISSUE #5324: [CAL-302] Installed Apps / Automation / Zapier - Zaps UI Issues
Currently there's a few differences from the Figma designs including

* Designs use a background behind the Zaps
* There should be no border around the logos
* The **Use Zap** button should be on a new line

Current

Should be

View in Figma

[App Store / Installed Apps / Workflow automation - Cal - Live (Figma)](https://www.figma.com/file/xk4HOxtSI82J0F7enMxeak/Cal---Live?node-id=161%3A96057)

> From [SyncLinear.com](https://synclinear.com)

### PR DESCRIPTION
**Pull request by om-ray**
## What does this PR do?

This PR changes the UI for Zap automation to match the Figma
Before
image

After
Screen Shot 2022-11-03 at 3 24 51 PM

Fixes #5324 

**Environment**: Staging(main branch) / Production

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: packages/app-store/zapier/components/TemplateCard.tsx [hunk 1]
```diff
@@ -17,11 +17,11 @@ export default function TemplateCard({ template }: { template: Template }) {
             <p className="mt-[2px] text-xs text-gray-500 ">{template.text}</p>
```
Comment (CarinaWolli) [inline packages/app-store/zapier/components/TemplateCard.tsx:17]: template.app should have `leading-4` and template text should be `text-sm `

**Review by CarinaWolli — CHANGES_REQUESTED** [review]
Looks good overall 🙌🏻 Just found some minor design issues.
In addition to the comments below: If one template card has more lines of text than the other one, the buttons don't have the correct position: 
Screenshot 2022-11-04 at 12 28 00

File: packages/app-store/zapier/components/TemplateCard.tsx [hunk 2]
```diff
@@ -4,10 +4,10 @@ import { Template } from "./AppSettings";
 
 export default function TemplateCard({ template }: { template: Template }) {
   return (
-    <div className="rounded-md border border-gray-200 p-4 ">
-      <div className="flex items-center justify-center">
+    <div className="rounded-md border border-gray-200 bg-white p-4">
```
Comment (CarinaWolli) [inline packages/app-store/zapier/components/TemplateCard.tsx:7]: should be `p-5`

**Comment by Jaibles** [comment]
Nothing to add on top of @CarinaWolli's feedback. Looks great 👍

File: packages/app-store/zapier/components/TemplateCard.tsx (hunk 1 above)
Comment (PeerRich) [reply packages/app-store/zapier/components/TemplateCard.tsx:17]: cc @om-ray

File: packages/app-store/zapier/components/TemplateCard.tsx (hunk 2 above)
Comment (PeerRich) [reply packages/app-store/zapier/components/TemplateCard.tsx:7]: @CarinaWolli you can also use the recommend feature of GitHub

File: packages/app-store/zapier/components/TemplateCard.tsx (hunk 1 above)
Comment (om-ray) [reply packages/app-store/zapier/components/TemplateCard.tsx:17]: I'll get this in right now!

**Comment by om-ray** [comment]
@CarinaWolli Oh! i didn't notice that at all, I'll fix that asap!

**Comment by om-ray** [comment]
@CarinaWolli It should be fixed now, thank you so much for your reviews!

**Review by CarinaWolli — APPROVED** [review]
Nice work! Thank you for contributing 🙏

**Pull request merged**